File: PSP/Proyectos/Login_DavidRoldan/src/main/java/domain/modelo/Order.java
package domain.modelo;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Order {
    private int id;
    private LocalDateTime date;
    private int customerid;
    private int tableid;
}

File: Móviles/Proyectos/Login_DavidRoldan/app/src/main/java/com/example/login_davidroldan/domain/modelo/Order.kt
package com.example.login_davidroldan.domain.modelo

import com.example.login_davidroldan.data.model.OrderResponse
import java.time.LocalDateTime

data class Order(
    val id: Int,
    val date: LocalDateTime,
    val customerId: Int,
    val tableId: Int,
)
fun Order.toOrderResponse() : OrderResponse = OrderResponse(id, date, customerId, tableId)
